### PR TITLE
Update server.php

### PR DESCRIPTION
**Pull request by shreegilliorkar**
While checking the valid password, the hash value of password in the database & user entered password must be checked.

### CODE REVIEW

File: source/server.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
   $result_find_student = mysqli_query($db,$query_find_student);
   if (mysqli_num_rows($result_find_student) == 1) {
     $row = mysqli_fetch_assoc($result_find_student);
-    if($password == $row['password']){
+    if($password == md5($row['password'])){
```
Comment (kmanadkat) [inline source/server.php:17]: The passwords in DB must be stored in MD5 hash format. No need to find hash of hash(password in row from db). Also notice `$password` is already a hashed password, computed on line 12.

File: source/server.php [hunk 2]
```diff
@@ -36,7 +36,7 @@
   $result_find_admin = mysqli_query($db,$query_find_admin);
   if (mysqli_num_rows($result_find_admin) == 1) {
     $row = mysqli_fetch_assoc($result_find_admin);
-    if($adminPassword == $row['password']){
+    if($adminPassword == md5($row['password'])){
```
Comment (kmanadkat) [inline source/server.php:39]: The passwords in DB must be stored in MD5 hash format. No need to find hash of hash(password in row from db). Also notice `$password` is already a hashed password, computed on line 12.

**Comment by kmanadkat** [comment]
Check this file: https://github.com/kmanadkat/housekeeper/blob/master/source/allothandler.php#L25
Here while registering student, password is first hashed and then stored into DB. Here the password by default is 12345 which is hardcoded, you may make it dynamic.

As far as I remember, there is no way to register admin from within project, you would have directly to add Admin entry in DB table. Even there, first hash the password (md5 format) using online tool and then save in DB table.

**Comment by shreegilliorkar** [comment]
okay, Got it now!

**Pull request closed without merging**